recipes/fenster/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.0.0.cci.20240909":
```

Comment:
Do you think we could ask upstream to tag a version at some point?

Reply:
I've just tagged the latest as `v0.1.0`, and hope to follow semantic versioning afterwards

Reply:
Thank you! :slightly_smiling_face: